.github/dependabot.yml

```diff
@@ -4,3 +4,6 @@ updates:
     directory: "/" # Location of package manifests
     schedule:
       interval: "weekly"
+    commit_message:
```

Comment:
It's it `commit-message` instead of `commit_message`?

See https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates#commit-message

Reply:
It is! my bad @VaiTon I'll open another PR